protocol/0035-LIQM-liquidity_monitoring.md

```diff
@@ -62,9 +62,28 @@ During the liquidity monitoring auction new or existing LPs can commit more stak
 ## What happens during the auction?
 
 The auction proceeds as usual. Please see the auction spec for details.
+## Frequency of checking for auction entry triggers
+
+ Through a series of actions which occur coincidently the market may be moved into a state in which a liquidity auction is expected and then back out of said state. Ideally, liquidity auctions should only be entered when the market truly requires one as once entered a minimum auction length must be observed. Even without a minimum auction length, a market flickering between two states is suboptimal. 
+
+ To resolve this, the triggers should only be checked at the end of each batch of transactions occurring with an identical timestamp (in the current Tendermint implementation this is equivalent to once per block). At the end of each such period the auction conditions should be checked and the market moved into liquidity auction state if the triggers have been hit. Similarly, the criteria for exiting the auction should be checked on the same frequency, so that a market cannot leave a liquidity auction only to immediately re-enter it at the end of the block.
+
+ A liquidity provider should still be unable to remove their stake if doing so would reduce the total stake below the target stake even though the market would remain open between the removal and the end of the block.
```

Comment:
```suggestion
 A liquidity provider amending LP provision order can reduce their stake (as long as total stake >= target stake) even if doing so would mean that at the end of block the system increases liquidity auction (because e.g. max open interest increased or because another LP was removed due to not meeting margin commitments). 
```